content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -556,3 +556,5 @@ You can expose a Service in multiple ways that don't directly involve the Ingres
 * Learn about the [Ingress API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#ingress-v1beta1-networking-k8s-io)
 * Learn about [Ingress controllers](/docs/concepts/services-networking/ingress-controllers/)
 * [Set up Ingress on Minikube with the NGINX Controller](/docs/tasks/access-application-cluster/ingress-minikube/)
+* Ingress is a top-level resource in the Kubernetes REST API.
```

Comment:
Hi @feloy . I searched for "/generated/kubernetes-api" under en/docs/concepts/. There are ~50 links that need to be updated. Could you update the links or do you have other ideas?
What do you think about modifying the existing links
From:
Line 556, ```Learn about the [Ingress API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#ingress-v1beta1-networking-k8s-io)```

To: 
```
Learn about the [Ingress API](/docs/reference/kubernetes-api/services-resources/ingress-v1/)
```

Reply:
Yes, I'll update these links.

I have planned to use the shortcode created in this PR instead:

```
Learn about the {{< api-reference page="services-resources/ingress-v1/" >}} API 
```

Reply:
I can see it is not possible to link directly to specific Resource or Definition on the page. For example:

- link to NodeStatus on this page: https://deploy-preview-23294--kubernetes-io-master-staging.netlify.app/docs/reference/kubernetes-api/cluster-resources/node-v1/

  The Anchor exists on the page, but it is not possible to specify the anchor on the api-reference shortcode. I will make changes to the shortcode to be able to specify an anchor, optionally

- link to ResourceRequirements on this page: https://deploy-preview-23294--kubernetes-io-master-staging.netlify.app/docs/reference/kubernetes-api/workloads-resources/container/

  The anchor does not exist on the page. I'll modify the generator to add anchors on such inlined definitions.

Reply:
Generator adds anchors for inlined definitions in this PR:
https://github.com/kubernetes-sigs/reference-docs/pull/186